pom.xml

```diff
@@ -137,7 +136,10 @@
                     <bin>${project.build.outputDirectory}</bin>
                     <srcs>
                         <src>${project.basedir}/src/org/rascalmpl/library</src>
+                        <!-- do not typecheck the checker or the tutor yet 
```

Comment:
the comment doesn't agree with what is happening. currently it is actually checking the tutor?

Reply:
On my screen the tutor is commented out